src/buildifier/buildifier_availability.ts

```diff
@@ -33,9 +35,15 @@ const BUILDTOOLS_RELEASES_URL =
  */
 export async function checkBuildifierIsAvailable() {
   const buildifierExecutable = getDefaultBuildifierExecutablePath();
+
   // Check if the program exists (in case it's an actual executable and not
   // an target name starting with `@`).
-  if (!buildifierExecutable.startsWith("@")) {
+  const isTarget = buildifierExecutable.startsWith("@")
+
+  // Check if the program exists as a relative path of the workspace
+  const isRelative = fs.existsSync(path.join(vscode.workspace.workspaceFolders?.[0]?.uri?.fsPath, buildifierExecutable));
```

Comment:
Afaict, `isRelative` will also be true for absolute paths, if that path is found? Maybe rename this to `foundWithoutPathEnv` or something similar?

Reply:
Good call, renamed